### PR TITLE
Feature: OpenAPI 3.0 docs

### PR DESCRIPTION
**Pull request by machisuji**
**IMPORTANT: Do _not_ squash.** _The history here is important as it contains the code for converting from API Blueprint to OpenAPI 3.0. We probably don't need it anymore. But if we do it would be unfortunate not to have it anymore._

Replaces API Blueprint docs (.apib files) with OpenAPI 3.0 ones (.yml), exposes spec under /api/v3/spec.json and adds swagger ui under /api/docs to try out the API.

We use the `openapi-cli` linter to validate the spec in a github workflow which should make sure no invalid spec is committed.
There are still 14 (mostly unknown types) errors (and a lot of warnings) present at this point, however. Fixing those is out-of-scope for this PR. Hence they've been added to `.redocly.lint-ignore.yaml` for now.

Either way the spec is good enough to be used by the integrated SwaggerUI. We still want to fix all errors and warnings eventually.

### CODE REVIEW

File: app/views/api_docs/index.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+FOOBAR
```
Comment (b12f) [inline app/views/api_docs/index.html.erb]: This file can be deleted I think

File: frontend/src/app/features/api-docs/swagger-ui/swagger-ui.component.html [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div id="docs-wrapper">
```
Comment (b12f) [inline frontend/src/app/features/api-docs/swagger-ui/swagger-ui.component.html]: Do you need the wrapper? I think you might even be able to have no template at all, and instead do something like

```
@Component({
  // other stuff
  // ...
  host: {
    id: 'swagger',
  },
})
```

Which would reduce both the complexity of the code but most importantly of the resulting html

**Review by b12f — COMMENTED** [review]
lgtm, a couple of small notes

File: frontend/src/app/features/api-docs/swagger-ui/swagger-ui.component.html (hunk 2 above)
Comment (machisuji) [reply frontend/src/app/features/api-docs/swagger-ui/swagger-ui.component.html]: Tried this but it doesn't seem to work. The compiler complains about a missing template. Leaving it as it is in that case.

**Pull request merged**